pkg/kubelet/dockershim/exec.go

```diff
@@ -110,29 +112,47 @@ func (*NativeExecHandler) ExecInContainer(client libdocker.Interface, container
 		return err
 	}
 
+	var execTimeout <-chan time.Time
```

Comment:
this needs to be only done when feature flag is set. Otherwise this will change the today's behavior.

Reply:
Good catch -- updated so that execTimeout is nil if the gate is not enabled.